package.json

```diff
@@ -54,6 +57,7 @@
     "react-scripts": "2.1.8",
     "react-sticky": "^6.0.3",
     "react-svg": "^7.1.0",
+    "react-test-renderer": "^16.8.6",
```

Comment:
Is it needed while we're using Enzyme?

Reply:
Yes, suggested from CRA website so i guess :)

Reply:
https://facebook.github.io/create-react-app/docs/running-tests#option-1-shallow-rendering